src/phases/filesystem/FlushingTemplate.yml

```diff
@@ -0,0 +1,1656 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This file defines a template to use to stress checkpointing, journal flush and dirty memory. The test itself is
+  simple:
+    - Run a single CrudActor targeting:
+      - test database, configurable via the Database parameter.
+      - Collection0 collection, configurable via the Collection parameter.
+      - 10 threads, configurable via the Threads parameter.
+      - 5000 iterations, configurable via the Iterations parameter.
+      - 1500 documents, configurable via the Documents parameter.
+      - 1 x 1000 bytes field in document, configurable via the DocumentSize parameter.
+      - {w:1} insert options, configurable via the Options parameter.
+  
+
+GlobalDefaults:
+  - rdIdExtraPaddingParam: &RecordIdExtraPaddingParam {^Parameter: {Name: "RecordIdExtraPadding", Default: 0}}
+  - PaddingGenerator: &PaddingGenerator {^FastRandomString: {length: *RecordIdExtraPaddingParam, alphabet: "a"}}
+  - Database: &database test
+  - Collection: &collection Collection0
+  - MaxPhases: &maxPhases 1
+  - InsertThreads: &insertThreads 10 # 10 # 25 # 100
+  - Iterations: &iterationsCount 5000
+  - DocumentSize: &documentSize {^Parameter: {Name: "DocumentSize", Default: 1000}}
+  - Options: &defaultOptions
+      WriteConcern:
+        Level: majority
+  - DefaultDocument: &defaultDocument  # The size of each document is about 1kb.
+      x: { ^FastRandomString: { length: *documentSize, alphabet: "x" } }
+      id: { ^Join: { array: [ { ^FormatString: { "format": "User%07d", "withArgs": [ { ^Inc: { start: 1 } } ] } }, *PaddingGenerator ] } }
+  - DefaultDocumentsList: &defaultDocumentList
```

Comment:
You can crib @SimonEismann 's use of ^FlattenArray to shorten this if you wanted.

Reply:
```
WriteOps10x: &WriteOps10x [
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                ]
  WriteOps100x: &WriteOps100x {^FlattenOnce: [
      *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
      *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
    ]}
```

Reply:
WriteOps10x: &WriteOps10x [
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                  {WriteCommand: insertOne, ...},
                ]
  WriteOps100x: &WriteOps100x {^FlattenOnce: [
      *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
      *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x, *WriteOps10x,
    ]}